pkg/test/framework/components/istio/kube.go

```diff
@@ -620,7 +620,6 @@ func commonInstallArgs(ctx resource.Context, cfg Config, c cluster.Cluster, defa
 
 	// Include all user-specified values and configuration options.
 	if cfg.EnableCNI {
-		args.AppendSet("components.cni.namespace", "kube-system")
```

Comment:
How would this affect current users who upgrade? If it was previously in kube-system, wouldn't they end up with the old version left behind (unless we properly document that CNI could be moved with this so they are aware they need to look in a different spot) and thus two competing CNIs?

Reply:
This is just the integration test framework default, shouldn't impact real users.

What annoys me is I have an existing install, and run the test locally and it breaks by deploying 2x.. maybe this is not a robust fix, but it does fix it

Reply:
Oops I missed the fact this is in test framework. :D

Reply:
This only impacts people running the test suite AFAIK, I don't think we have upgrade guarantees for the test suite, or any guarantees about running them on top of an existing install.